setup.py

```diff
@@ -15,7 +15,9 @@
 
 install_requires = [
     'Click>=6.0',
-    'cookiecutter>=1.6.0',
+    # We need the functionality provided by this branch. See this PR for more info:
+    # https://github.com/cookiecutter/cookiecutter/pull/907
+    'cookiecutter @ https://github.com/zillow/cookiecutter/tarball/strip_dir',
```

Comment:
hm, this hasn't been merged, do we want to rely on this? Will this be installed from github repo directly as a dependency?